nservicebus/messaging/databus/index.md

```diff
@@ -95,16 +95,17 @@ Automatically removing these attachments can cause problems in many situations.
 
 ## Alternatives
 
-- Use a different transport or a different transport tier
-- Message Compression: Use message body compression which works well on text-based payloads like XML and Json or any payload (text or binary) that contains repetitive data
-  - [Message mutator example demonstrating message body compression](/samples/messagemutators/)
-- Stream-based properties: The [Handling large stream properties via pipeline](/samples/pipeline/stream-properties/) sample demonstrates a purely stream-based approach (rather than loading the full payload into memory) implemented by leveraging the NServiceBus pipeline.
-- Binary Serializer: A binary serializer is more efficient and most serializers can be added with a few lines of code
-   - Some binary [serializers are maintained by the community](/nservicebus/community/#serializers)
-- Attachments: When dealing with unbounded binary payloads consider the [community maintained NServiceBus.Attachments](/nservicebus/community/#nservicebus-attachments)
-  - Read on demand: Will only retrieve attachment data when the consumer reads it
-  - Reduced Memory usage: No base64 serializer overhead resulting in a significant reduction in resource utilization
-- Use any of the above in combination with compression
+NOTE: A combination of these techniques may be used.
+
+- Use a different transport or different tier (e.g. Azure Service Bus _Premium_ instead of _Standard_).
+- Message compression: this works well on text-based payloads like XML and JSON, or any payload (text or binary) that contains repetitive data.
+  - See the [message mutator example demonstrating message body compression](/samples/messagemutators/).
+- Stream-based properties: The sample showing [handling large stream properties via pipeline](/samples/pipeline/stream-properties/) demonstrates a purely stream-based approach (rather than loading the full payload into memory) using the NServiceBus pipeline.
+- A more efficient serializer: For example, a binary serializer. Most serializers can be added with a few lines of code.
+   - Several [serializers are maintained by the community](/nservicebus/community/#serializers).
+- Attachments: the community maintained [NServiceBus.Attachments](/nservicebus/community/#nservicebus-attachments) package may be used for unbounded binary payloads.
+  - Read on demand: attachments are only retrieved when read by a consumer.
```

Comment:
```suggestion
  - Read on demand: Attachments are only retrieved when read by a consumer. Moreover, it processes all data items via an IAsyncEnumerable
```

Reply:
OK, here we have two sentences so a capital is warranted. I'll consider the list as a whole.